plugins/vst_base/RemoteVstPlugin.cpp

```diff
@@ -119,6 +119,29 @@ RemoteVstPlugin * __plugin = NULL;
 HWND __MessageHwnd = NULL;
 
 
+static std::wstring toWString(std::string s)
+{
+	/*
+	QString qstr = QString::fromUtf8(s.data(), int(s.size()));
```

Comment:
Beacause it requires including Qt headers(on Windows). If that don't matter (we already do that), I can switch to QString is needed.

Reply:
Ah, I see. @lukas-w has offered me advice on where it's okay to use Qt stuff in the past. I think things like QString, at least, are fine everywhere, but hopefully he'll chime in here to confirm that this is OK if he's not too busy.

Reply:
To be more clear, `RemoteVstPlugin` use Qt libraries for IPC on Windows. So I think that would be fine.

Reply:
@PhysSong is right, this is not a good place to use Qt because that would add MinGW Qt as a dependency on Linux builds. We can however use C++11 functions to convert to UTF-16:
```c++
#include <locale>
#include <codecvt>
```
```c++
std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
return converter.from_bytes(s);
```

This does the same thing, just in fewer lines. I think we already use C++11 for `RemoteVstPlugin` on `stable-1.2`, but I didn't check.

Reply:
> because that would add MinGW Qt as a dependency on Linux builds.

It seems like we can use host version Qt, but I don't think it's a good idea because it adds 32bit Qt as a dependency on 64bit Linux.

> We can however use C++11 functions to convert to UTF-16:

That doesn't work with our old MinGW because of missing header files(it should work with newest `g++-mingw-w64`). Those functions are deprecated in C++17, too.